drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,69 +1,114 @@
-function DragAndDropBlock(runtime, element) {
-
+function DragAndDropBlock(runtime, element, configuration) {
+    "use strict";
+    // Ensure "undefined" has not been redefined (though this unlikely and often impossible).
```

Comment:
`if (typeof blah.blah === 'undefined')` can be replaced with `if (blah.blah === undefined)`, but `if (typeof blah === 'undefined')` is not entirely equivalent to `if (blah === undefined)`, because the second form will fail with "blah is not defined" if the `blah` variable hasn't been defined anywhere.

I usually prefer `if (!window.gettext)`, though.